platform-includes/troubleshooting/dotnet.mdx

```diff
@@ -50,3 +50,11 @@ using SentrySession = Sentry.Session;
 ```
 
 Then `SentrySession` can be used instead of `Sentry.Session`.
+
+### Visual Studio Hot Restart Incompatibility
+
+Attempting to build a .NET MAUI iOS app will result in an error if you use Visual Studio Hot Restart when developing on Windows with a remote Mac:
+
+> The xcframework sentry.bindings.cocoa\5.0.1\lib\net8.0-ios17.0\Sentry.Bindings.Cocoa.resources.zip has an incorrect or unknown format and cannot be processed.
+
+This happens because Hot Restart does not support static iOS libraries or frameworks containing static libraries, as detailed in the [Microsoft documentation](https://learn.microsoft.com/en-us/dotnet/maui/ios/hot-restart?view=net-maui-9.0#limitations).
```

Comment:
@aritchie curious, nothing we can do about this? 

People need to uninstall Sentry to use Hot Restart?

Reply:
@bruno-garcia We may be able to detect hotrestart in a prebuild step, but I'm not sure.  I'll look into it.  I wish the tooling gave better error messages in these cases

Reply:
Just seems like we at least need to say then: "To use Hot Restart you'll need to remove the Sentry SDK" if that's the solution. But that's quite sub optimal obviously

Reply:
There is likely some form of condition that can be applied to the package reference and if/def constant you can use in code to remove UseSentry.  I'll ask around.

Thoughts

```xml
<PackageReference Include="Sentry.Maui" Condition="$(HOT_RESTART) == 'false'" />
```

```csharp
#if !HOT_RESTART
.UseSentry(...)
#endif
```